src/components/ChatEntry.jsx

```diff
@@ -1,20 +1,39 @@
 import './ChatEntry.css';
+import PropTypes from 'prop-types';
+import {useState} from 'react';
+import TimeStamp from './TimeStamp.jsx';
+
+const ChatEntry = ({sender,body,timeStamp, liked, numOfLikes, setLikes, isLocal}) => {
+  const [like, setLike] = useState(liked);
 
-const ChatEntry = () => {
   return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+    <div className={ `${isLocal === true ? 'local' : ''} chat-entry`}>
+      <h2 className="entry-name">{sender}</h2>
       <section className="entry-bubble">
-        <p>Replace with body of ChatEntry</p>
-        <p className="entry-time">Replace with TimeStamp component</p>
-        <button className="like">🤍</button>
+        <p>{body}</p>
+        <p className="entry-time"><TimeStamp time={timeStamp}/></p>
+        <button className="like" onClick={() =>{
+          if (like === true) {
+            setLike(false);
+            setLikes(numOfLikes - 1);
+          } else {
+            setLike(true);
+            setLikes(numOfLikes + 1);
+          }
```

Comment:
Instead of onClick being an anonymous function with logic, I'd expect the onClick event handler to invoke a function that is passed down from App.jsx to ChatLog.jsx and finally down to this component here ChatEntry.jsx. 

You can find an example of what this looks like at this [link](https://github.com/Ada-C22/flasky-front-end/commit/db8e1afd6d2e4cadf627841adcb1baec55544857#diff-d274a54187c91ba0f532df2a9e194e27ab50e988f5e4c33f5a7893918320c661), see lines 17-21 and line 35 in Cat.jsx

Petting a cat in flasky-front-end is the equivalent of liking a ChatEntry.